samples/snippets/speech_model_adaptation_beta_test.py

```diff
@@ -0,0 +1,29 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import google.auth
+
+import speech_model_adaptation_beta
+
+
+STORAGE_URI = "gs://cloud-samples-data/speech/brooklyn_bridge.raw"
+_, PROJECT_ID = google.auth.default()
+LOCATION = "us-west1"
+
+
+def test_model_adaptation_beta(capsys):
+    transcript = speech_model_adaptation_beta.transcribe_with_model_adaptation(
```

Comment:
Please add fixtures for the custom class and phrase set. Each fixture should generate a unique name with a UUID,  yield the name, and then delete the custom class and phrase set.

```py
@pytest.fixture
def custom_class_id():
    custom_class_id = f"test_custom_class_{uuid.uuid4()}"
    yield custom_class_id
    # call API method to delete this custom_class
    ...

@pytest.fixture
def phrase_set_id():
    phrase_set_id = f"test_phrase_set_{uuid.uuid4()}"
    yield phrase_set_id
    # call API method to delete this phrase set
    ...
``` 

These fixtures can than be passed through to the sample function.

This is a good example of how to incorporate a fixture into a sample test. https://github.com/GoogleCloudPlatform/python-docs-samples/blob/89e67bcb655ed6d120dee46d2d40d866f7c72a59/storage/cloud-client/acl_test.py#L59-L65